### PR TITLE
Add disclaimer next to message textarea in feedback form

### LINKED ISSUE #12543: Add disclaimer next to message textarea in feedback form
This disclaimer should be added to the feedback form message (which is and needs to stay optional):

> Please provide any additional information that may help us to understand your report (including which policy you believe has been violated).  While this information is not required, failure to include it may prevent us from addressing the reported content.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDFRNT-262)

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #12543

---

image

### CODE REVIEW

**Comment by willdurand** [comment]
(please only review the last commit)

File: src/amo/components/FeedbackForm/index.js [hunk 1]
```diff
@@ -383,6 +383,13 @@ export class FeedbackFormBase extends React.Component<InternalProps, State> {
                 onChange={this.onFieldChange}
                 value={this.state.text}
               />
+              <p className="FeedbackForm--help">
+                {i18n.gettext(`Please provide any additional information that
```
Comment (diox) [inline src/amo/components/FeedbackForm/index.js:387]: I don't think we should be breaking this text over several lines like that ?

File: src/amo/components/FeedbackForm/styles.scss [hunk 2]
```diff
@@ -71,8 +71,12 @@ input[type='checkbox'] + .FeedbackForm--label {
 .FeedbackForm--help {
   color: $grey-50;
   display: block;
-  font-size: $font-size-xs;
-  margin-top: 4px;
+  margin-top: 0;
+  max-width: 700px;
```
Comment (diox) [inline src/amo/components/FeedbackForm/styles.scss:75]: I don't think we should be breaking this text over several lines like that ?

Comment (willdurand) [reply src/amo/components/FeedbackForm/styles.scss:75]: That's coming from UX because the text is too long otherwise (and still multi-line).

Comment (diox) [reply src/amo/components/FeedbackForm/styles.scss:75]: mmm I think it looks quite weird given the fields themselves (message, but also the location select) are full width...

Comment (willdurand) [reply src/amo/components/FeedbackForm/styles.scss:75]: ok, I revert for now..

**Review by diox — APPROVED** [review]

**Pull request merged**